foundationdb/tox.ini

```diff
@@ -0,0 +1,24 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py38
+envlist =
+    py{27,38}
```

Comment:
```suggestion
    py38
```

Reply:
Hm, does this mean we're dropping support for 2.7? (Fine for us, just want to understand what it's doing.)

Reply:
Yup! We only require Python 3 for this.